### PR TITLE
[13.0][MIG] migration of module sale_operating_unit

### PR DESCRIPTION
**Pull request by alan196**
Depends on https://github.com/OCA/operating-unit/pull/229

### CODE REVIEW

File: sale_operating_unit/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
+=======================
```
Comment (newtratip) [inline sale_operating_unit/README.rst:1]: Could you generate readme to 13.0 ?

File: sale_operating_unit/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                    _(
+                        "Configuration error. The Company in "
+                        "the Sales Order and in the Operating "
+                        "Unit must be the same."
+                    )
+                )
+
+    def _prepare_invoice(self):
+        self.ensure_one()
+        invoice_vals = super(SaleOrder, self)._prepare_invoice()
```
Comment (newtratip) [inline sale_operating_unit/models/sale_order.py:67]:
```suggestion
        invoice_vals = super()._prepare_invoice()
```

File: sale_operating_unit/report/sale_report.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+
+    _inherit = "sale.report"
+
+    operating_unit_id = fields.Many2one("operating.unit", "Operating Unit")
```
Comment (newtratip) [inline sale_operating_unit/report/sale_report.py:10]:
```suggestion
    operating_unit_id = fields.Many2one(comodel_name="operating.unit", string="Operating Unit")
```

File: sale_operating_unit/report/sale_report.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+    def _query(self, with_clause="", fields=False, groupby="", from_clause=""):  # noqa
+        if not fields:
+            fields = {}
+        fields["operating_unit_id"] = ", s.operating_unit_id as operating_unit_id"
+        groupby += ", s.operating_unit_id"
+        return super(SaleReport, self)._query(with_clause, fields, groupby, from_clause)
```
Comment (newtratip) [inline sale_operating_unit/report/sale_report.py:17]:
```suggestion
        return super()._query(with_clause, fields, groupby, from_clause)
```

File: sale_operating_unit/wizard/sale_make_invoice_advance.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+from odoo import models
+
+
+class SaleAdvancePaymentInv(models.TransientModel):
+    _inherit = "sale.advance.payment.inv"
+
+    def _create_invoice(self, order, so_line, amount):
+        invoice = super(SaleAdvancePaymentInv, self)._create_invoice(
```
Comment (newtratip) [inline sale_operating_unit/wizard/sale_make_invoice_advance.py:8]:
```suggestion
        invoice = super()._create_invoice(
```

**Review by newtratip — APPROVED** [review]
Looks good to me, waiting #229 merged :+1:

**Review by lmgr0312 — APPROVED** [review]
LGTM!

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional test

**Comment by alan196** [comment]
@AaronHForgeFlow I just rebased with the latest commit

**Pull request merged**